ui/src/shared/components/flux_functions_toolbar/FluxFunctionsToolbar.tsx

```diff
@@ -71,10 +71,14 @@ class FluxFunctionsToolbar extends PureComponent<Props, State> {
     this.setState({searchTerm})
   }
 
-  private handleUpdateScript = (funcExample: string) => {
+  private handleUpdateScript = (funcName: string, funcExample: string) => {
     const {activeQueryText, onSetActiveQueryText} = this.props
 
-    onSetActiveQueryText(`${activeQueryText}\n  |> ${funcExample}`)
+    if (funcName === 'from') {
```

Comment:
How about:
```suggestion
    if (funcName.startsWith('from')) {
```
This way it will handle the `fromRows` function too, which I think also should not have a pipe prepended. I think there will be other similar `from` functions in the future, like `fromCSV`, `fromSQL`, etc.

Reply:
Okay, makes sense! Thanks :)